packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -154,6 +156,12 @@ function FastBridgesTable() {
   )
 }
 
+function getCalendarUrl(confirmationDays: number) {
+  const title = 'Arbitrum Withdrawal'
```

Comment:
Might be worth being a bit more specific here in case people are withdrawing different assets from different chains, so something like `0.5 ETH Withdrawal from Arbitrum Nova`.